deploy/playbook.yml

```
@@ -0,0 +1,73 @@
+- name: Install to-do app in module4 servers
+  hosts: module4
+  vars_prompt:
+    - name: trello_api_key
+    - name: trello_token
+
+  tasks:
+  - name: Install git
+    ansible.builtin.yum:
+      name: git
+      state: latest
+    become: true  
+
+  - name: Install python 3.x
+    ansible.builtin.yum:
+      name: python3
+      state: latest
+    become: true  
```

Minor, but if you're using the same state then it's easy to combine these:
```suggestion
  - name: Install git & Python3
    ansible.builtin.yum:
      name:
        - git
        - python3
      state: latest
    become: true  
```

That's a bit more concise, but I'll let you judge whether you think it's cleaner!

---

Thanks for the tip, did not consider this could be done.